.taskcluster.yml

```diff
@@ -53,6 +54,39 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("test_mozilla_central")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          env:
+            mozilla_central: gecko-dev
+            mozilla_central_output: /tmp/mozilla_central_output
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
+               [ ! -d ${mozilla_central} ] &&
```

Comment:
You can do either: clone if the dir doesn't exist, cd & pull if the dir exists, or clone ignoring errors, then cd & pull.